src/main/scala/io/iohk/ethereum/db/storage/BlockHeadersNumbersStorage.scala

```diff
@@ -0,0 +1,17 @@
+package io.iohk.ethereum.db.storage
+
+import akka.util.ByteString
+import io.iohk.ethereum.db.dataSource.DataSource
+
+class BlockHeadersNumbersStorage(val dataSource: DataSource) extends KeyValueStorage[BigInt, ByteString] {
```

Comment:
I've already have made this in https://github.com/input-output-hk/etc-client/pull/51/files#diff-675d2b192a8da56fd973c3416cb2b4b0 :cry: